compiler/AST/build.cpp

```diff
@@ -264,8 +264,34 @@ Expr* buildIntLiteral(const char* pch, const char* file, int line) {
     ull = octStr2uint64(pch, true, file, line);
   else if (!strncmp("0x", pch, 2) || !strncmp("0X", pch, 2))
     ull = hexStr2uint64(pch, true, file, line);
-  else
-    ull = str2uint64(pch, true, file, line);
+  else {
+    int j = 0;
+    int len = strlen(pch);
+    char noUnderscores[len+1];
```

Comment:
I can never remember if C run-time length arrays are legal in standard C or if they are an extension. In any case I'd probably just malloc the result here. If you keep the variable length array, you really really need to check that len isn't say 2GB.

Reply:
Good call. I just updated it to use malloc/free instead.